app/assets/javascripts/components/ResponseView.coffee

```diff
@@ -32,28 +29,17 @@
   displayName: 'CreationDate'
 
   render: ->
-    { createdAt } = @props
+    { requestedAt, createdAt } = @props
 
     div
       className: 'creation-date'
       i
         className: 'fa fa-clock-o fa-lg'
-      "Ingevuld op: #{moment(createdAt).format('dddd D-M-Y H:mm')}"
-
-@EstimateAndVariance = React.createClass
-  displayName: 'EstimateAndVariance'
-
-  render: ->
-    { estimate, variance } = @props
-
-    div
-      className: 'estimate-and-variance'
-      div
-        className: 'estimate'
-        "Schatting: #{estimate}"
-      div
-        className: 'variance'
-        "Variantie: #{variance}"
+      span {},
+        "Aangevraagd op: #{moment(requestedAt).format('dddd D-M-Y H:mm')}",
+        br {}
+      span {},
+        "Ingevuld op: #{moment(createdAt).format('dddd D-M-Y H:mm')}"
```

Comment:
Omdat de moment-call nu op meerdere plekken wordt gedupliceerd, mag het van mij wel naar een helper method, bijvoorbeeld in app/assets/javascripts/global_helpers.coffee